Assets/Css/style.css

```diff
@@ -14,55 +14,6 @@ body {
   background-size: cover;
   font-family: "Poppins", sans-serif;
 }
-.options{
```

Comment:
Please revert these deletion of CSS as they are supposed to be there.bcz of this CSS of the button in Get help section is effected